Fix: Clarify behavior of HTML tags within the <title> element (Issue #36702) (#36827)
mdn/content: files/en-us/web/api/htmltitleelement/text/index.md

{{APIRef("HTML DOM")}}

> The **`text`** property of the {{domxref("HTMLTitleElement")}} interface represents the child text content of the document's title as a string. It contains the {{HTMLelement("title")}} element's content as text; if HTML tags are included within the `<title>` element, they are included as part of the string value rather than being parsed as HTML.

Reviewer: ```suggestion
The **`text`** property of the {{domxref("HTMLTitleElement")}} interface represents the child text content of the document's {{htmlelement("title")}} as a string. It contains the {{HTMLelement("title")}} element's content as text; if HTML tags are included within the `<title>` element, they are included as part of the string value rather than being parsed as HTML.
```

```js
const title = document.querySelector("title");

> console.log(title.text); // yield: "Hello world! <span class="highlight">Isn't this wonderful</span> really?"

Reviewer: ```suggestion
console.log(title.text); // "Hello world! <span class="highlight">Isn't this wonderful</span> really?"
title.text = "Update the title";
```
let's add a setter example

```

> As you can see, the `span` tag and its content were treated as plain text and displayed exactly as they appear in the `title` element, rather than being processed as HTML.

Reviewer: ```suggestion
As you can see, the `span` tag remained unparsed; the `<title>` element's contents were treated as plain text and returned exactly as they appear in the `title` element.
```
